content/use-case-apps.md

```diff
@@ -0,0 +1,252 @@
+---
+title: "There's an Apps for that"
+description: Endless possibilities with Kestra Apps
+date: 2024-12-12T13:00:00
+category: Solutions
+author:
+  name: Benoit Pimpaud
+  image: bpimpaud
+image: /blogs/introducing-apps.jpg
+---
+
+We recently introduced **Apps**: **frontend applications** for your Kestra workflows. They allow end-users to interact with workflows through forms, output displays, markdown blocks, approval buttons, and other UI components, while Kestra flows handle all backend processing.
+
+
+
+## Requests & Review
+
+One use case we see a lot among Kestra customers is the need for ingesting data into FTP. Uploading Excel file in FTP server is still something very common, even in the era of Gen AI.
```

Comment:
```suggestion
Uploading files to an FTP server—still a thing, right? But doing it securely and efficiently? That’s where the pain begins.

Kestra Apps simplify it all. Instead of dealing with credentials, server configurations, or outdated UIs, users can select their FTP configuration, choose a folder, and upload their file with a single click.

Automation shouldn’t feel manual. With Apps, workflows handle the complexity while users get an experience that makes sense.
```

Reply:
applied in branch